js/packages/teams-ai/package.json

```diff
@@ -34,12 +34,18 @@
         "botbuilder": "^4.19.3",
         "botbuilder-dialogs": "^4.19.3",
         "gpt-3-encoder": "^1.1.4",
+        "jwt-decode": "^3.1.2",
+        "uuid": "^9.0.0",
         "yaml": "^2.2.2"
     },
+    "devDependencies": {
+        "@types/node": "^20.4.2",
+        "@types/uuid": "^9.0.2",
+        "typescript": "^5.1.6"
```

Comment:
These should already be at root. Can we leave this out and depend on the monorepo pattern?